Cargo.toml

```diff
@@ -13,3 +13,8 @@ edition = "2018"
 
 [dev-dependencies]
 rand = "0.8.3"
+criterion = "0.3"
+
+[[bench]]
+name = "criterion_benchmarks"
```

Comment:
Let's not name the benchmarks using the name of the library that runs them. I'd actually suggest that we split the benchmarks into two - `distance_benchmark` and `edit_benchmark` just to separate things better. If this seems like overkill to you, perhaps consider renaming it to `levenshtein_diff_benchmarks` or something?

Reply:
I don't think it makes sense to separate the benchmarks at this point. Maybe in the future if more are added then it could make more sense. You can selectively run individual benchmarks using `Cargo bench -- generate_edits`.

You can't combine benchmarks that use two separate harnesses (at least not easily), so if I added Iai benchmarks later I was planning to put them in a file called `iai_benchmarks.rs`. I'm open to other names though. I will try to see how other projects have organized things.

Reply:
I still think naming your suite of benchmarks based on the library you use to run them doesn't make a lot of sense. It might make sense if and when we add the Iai benchmarks though, and in that case, the PR that does that will be the right place to [re]name files.

It is a good idea to research how other projects structure things, especially since the author of the two libraries Criterion and Iai says that they're meant to be cooperative (so I assume quite a few projects will be using both). If naming the benchmarks based on the names of the libraries that run them is a convention, I am fine with moving forward with it. Otherwise, I am not very sure.

Reply:
I decided to rename the file to `benchmarks.rs`. There wasn't much consistency to be found looking at other projects. There's a lot of different ways to set things up with Cargo. Some projects even had the benchmarks built as its own crate for example. The benchmark suite is small enough right now that I think this is _fine_.

Reply:
Cool, that works for me.